spec/spec_helper.rb

```diff
@@ -3,10 +3,13 @@
 require 'equivalent-xml'
 require 'securerandom'
 require 'timecop'
+require 'webmock/rspec'
 
 # Load all files from `spec/support`
 Dir[File.dirname(__FILE__) + "/support/**/*.rb"].each {|f| require f }
 
+WebMock.disable!
```

Comment:
Why `.disable!` instead of just  `.disable_net_connect`. The former seems to more comprehensively disable different network adapters, the later seems to just disable external network connects. Although as far as we care that may result in the same thing... Mostly just curious. Was looking at the code and not too sure what the result is expected to be: https://github.com/bblimke/webmock/blob/e386be1cb3652c83d76fec57847ea4bed581b992/lib/webmock/webmock.rb

Reply:
yea my bad. will fix that